src/help/zaphelp/contents/ui/dialogs/options/checkforupdates.html

```diff
@@ -46,8 +46,8 @@
 They may be incomplete, contain significant issues or cause stability problems.
 
 <H3>Add-on directories</H3>
-ZAP will load all of the add-ons located in the release 'plugins' directory as well as any that have been downloaded
-and are in the local 'plugins' directory.<br>
+ZAP will load all of the add-ons located in the release 'plugin' directory as well as any that have been downloaded
+and are in the local 'plugin' directory.<br>
```

Comment:
Pointing to the other page simplifies things.

For my 2cents, install makes sense and has an obvious implication from an end user perspective but I'm not sure default does.